### PR TITLE
Change wait time multiplier mechanism

### PR DESCRIPTION
**Pull request by ilikesymmetry**
Rids us of needing a column for each multiplier we have in the table

### CODE REVIEW

File: src/eatery/static_eatery.py [hunk 1]
```diff
@@ -115,4 +115,4 @@ def string_to_date_range(dates):
   start_str, end_str = dates.split('-')
   start_date = datetime.strptime(start_str, '%m/%d/%y').date()
   end_date = datetime.strptime(end_str, '%m/%d/%y').date()
-  return [start_date, end_date]
+  return (start_date, end_date)
```
Comment (kq-li) [inline src/eatery/static_eatery.py:118]: i'd personally remove the parens, @young-k?

**Review by kq-li — APPROVED** [review]
tight

**Review by meganle — APPROVED** [review]
Good catch 🎣

File: src/eatery/static_eatery.py (hunk 1 above)
Comment (young-k) [reply src/eatery/static_eatery.py:118]: i don't really mind either way, up to @meganle

**Pull request merged**